pkg/certificates/bundle.go

```diff
@@ -0,0 +1,101 @@
+package certificates
```

Comment:
Please rename this to certificates.go

unless a new complex object is being introduced we should try to keep file names == package names